x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -234,6 +239,9 @@ export interface Installation extends SavedObjectAttributes {
   es_index_patterns: Record<string, string>;
   name: string;
   version: string;
+  install_status: EpmPackageInstallStatus;
+  install_version: string;
+  install_started_at: string;
```

Comment:
After a successful installation, having

```
install_status: 'installed',
install_started_at: Date
```

feels incomplete. `install_started_at` doesn't seem useful any longer. At least not by itself.

I think I'd expect something like `install_ended_at` to capture the install duration

```
install_status: 'installed',
install_started_at: Date
install_ended_at: Date
```

Or `install_updated_at` which works for both installed & installing

```
install_status: 'installed',
install_updated_at: Date
```

```
install_status: 'installing',
install_updated_at: Date
```

Reply:
I guess there just wasn't a need for `install_ended_at` currently, otherwise we could have that at some point.  This case is for packages that never finished installing for some reason.

`install_updated_at` is good, I'm just worried about whether that would confuse people into thinking it was the last time the installation was updated, which is not true, as it gets updated multiple times as the installation progresses.